src/test/java/apicurito/tests/steps/verification/MainPageVerifications.java

```diff
@@ -54,28 +55,29 @@ public void checkThatAPIProduce(String expectedProduce) {       //NEW
     }
 
     @Then("^check that API contact info is$")
-    public void checkThatAPIContactInfoIs(DataTable table) {
+    public void checkThatAPIContactInfoIs(DataTable table) {        //NEW
         CommonUtils.getAppRoot().$("title-bar").$("h1").click();
         for (List<String> dataRow : table.cells()) {
             String name = MainPageUtils.getMainPageRoot().$("contact-section").$(By.className("name")).getText();
+            assertThat(name).as("Checking API contatct name:").isEqualTo(dataRow.get(0));
+
             String email = MainPageUtils.getMainPageRoot().$("contact-section").$(By.className("email")).getText();
-            String url = MainPageUtils.getMainPageRoot().$("contact-section").$(By.className("url")).getText();
+            assertThat(email).as("Checking API contatct email:").isEqualTo(dataRow.get(1));
 
-            CollectorHelper.getCollector().assertThat(name).as("Checking API contatct name:").isEqualTo(dataRow.get(0));
-            CollectorHelper.getCollector().assertThat(email).as("Checking API contatct email:").isEqualTo(dataRow.get(1));
-            CollectorHelper.getCollector().assertThat(url).as("Checking API contatct url:").isEqualTo(dataRow.get(2));
+            String url = MainPageUtils.getMainPageRoot().$("contact-section").$(By.className("url")).getText();
```

Comment:
Do you plan to add inside class with all element selectors as we discussed?

Reply:
yes, I have already created a jira task.

Reply:
ok